File: mcached/lua/create/mcached_create.lua
#! /usr/bin/lua
-- ----------------------------------------------------
--	mcached_create.lua
--
--				Feb/16/2015
--
-- ----------------------------------------------------
json=require	('json')
require	('text_manipulate')
require	('mcached_manipulate')

-- ----------------------------------------------------
function data_prepare_proc ()
	dict_aa = {}

	dict_aa = dict_append_proc (dict_aa,"t1731","金沢",83275,"1981-5-17")
	dict_aa = dict_append_proc (dict_aa,"t1732","輪島",47954,"1981-8-9")
	dict_aa = dict_append_proc (dict_aa,"t1733","小松",58743,"1981-9-15")
	dict_aa = dict_append_proc (dict_aa,"t1734","七尾",41329,"1981-3-22")
	dict_aa = dict_append_proc (dict_aa,"t1735","珠洲",86178,"1981-7-11")
	dict_aa = dict_append_proc (dict_aa,"t1736","加賀",38527,"1981-9-21")
	dict_aa = dict_append_proc (dict_aa,"t1737","羽咋",49536,"1981-1-4")
	dict_aa = dict_append_proc (dict_aa,"t1738","かほく",27841,"1981-10-21")
	dict_aa = dict_append_proc (dict_aa,"t1739","白山",45386,"1981-3-18")

	return	dict_aa
end
-- ----------------------------------------------------
print ("*** 開始 ***")

dict_aa = data_prepare_proc ()

local server_host = "localhost"
local server_port = 11211

dict_to_mcached_proc (server_host,server_port,dict_aa)

print ("*** 終了 ***")
-- ----------------------------------------------------

File: mcached/node/create/mcached_create.js
#! /usr/bin/nodejs
// ---------------------------------------------------------------
//	mcached_create.js
//
//					Feb/11/2016
//
// ---------------------------------------------------------------
var memcache = require("memcache")
var text_manipulate=require ("/var/www/data_base/common/node_common/text_manipulate")

// ---------------------------------------------------------------
function data_prepare_proc ()
{
	var dict_aa = new Object ()

	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1731','金沢',59784,'1950-5-12')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1732','輪島',48157,'1950-4-15')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1733','小松',93241,'1950-10-2')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1734','七尾',31864,'1950-6-22')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1735','珠洲',42358,'1950-8-14')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1736','加賀',65192,'1950-9-12')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1737','羽咋',37251,'1950-3-21')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1738','かほく',52486,'1950-7-26')
	dict_aa = text_manipulate.dict_append_proc (dict_aa,'t1739','白山',57391,'1950-10-2')
	
	return	dict_aa
}

// ---------------------------------------------------------------
function set_proc (client,key,value)
{
//	client.set(key, value,function(error, result){ }, 3600)
	client.set (key,value,function(error, result){ })
}

// ---------------------------------------------------------------
var onConnect = function()
{
	console.log ("*** 開始 ***")

	var dict_aa = data_prepare_proc ()

	for (var key in dict_aa)
		{
		var unit_aa = dict_aa[key]
		var str_unit = JSON.stringify (unit_aa)
		console.log (str_unit)
		set_proc (client,key,str_unit)
		}

	client.close()

	console.log ("*** 終了 ***")
}

// ---------------------------------------------------------------
var client = new memcache.Client (11211,'localhost')
client.connect()
client.addHandler(onConnect)

// ---------------------------------------------------------------
